jbpm-services/jbpm-services-api/src/main/java/org/jbpm/services/api/query/QueryResultMapper.java

```diff
@@ -98,6 +99,17 @@
     public static final String COLUMN_WORKITEMID = "WORKITEMID";
     public static final String COLUMN_ORGANIZATIONAL_ENTITY = "ID";
     public static final String COLUMN_EXCLUDED_OWNER = "ENTITY_ID";
+    public static final String COLUMN_EXPIRATIONTIME = "EXPIRATIONTIME";
+    public static final String COLUMN_POTOWNER = "POTOWNER";
+    public static final String COLUMN_TASK_TYPE = "TASKTYPE";
+    public static final String COLUMN_FORM_NAME = "FORMNAME";
+    public static final String COLUMN_SUMMARY = "SUMMARY";
+    public static final String COLUMN_OPTLOCK = "OPTLOCK";
```

Comment:
potluck is an internal hibernate thing so should not really be returned as part of the queries.

Reply:
Ok Maciej. The customer would like to see the optlock in the queries results. But I remove it and I talk to them.

Reply:
Done.